scan-command/src/test/resources/command-outputs/unix/print-rules-to-console.txt

```diff
@@ -2,6 +2,7 @@ Loading scan tool configurations from src/test/resources/test-resources/bal-proj
 	RuleID                                           | Rule Kind     | Rule Description         
 	---------------------------------------------------------------------------------------------
 	ballerina:1                                      | CODE_SMELL    | Avoid checkpanic
+	ballerina:10                                     | CODE_SMELL    | Invalid range operator
```

Comment:
This is wrong, isn't it? 10 should come after 2.

Reply:
Yes, there is an already a issue.
It will fix on that https://github.com/ballerina-platform/static-code-analysis-tool/issues/42